csharp-basics/exercises/Arrays/Arrays/TicTacToe/Program.cs

```diff
@@ -10,6 +10,37 @@ private static void Main(string[] args)
         {
             InitBoard();
             DisplayBoard();
+            
+            bool turnOfX = false; ;
+            int row, column;
+            string messageToUser;
+            do
+            {
+                turnOfX = !turnOfX;
+                do
+                {
+                    messageToUser = "";
+                    if (turnOfX) Console.WriteLine("X choose Your location");
+                    else Console.WriteLine("O choose Your location");
+                    Console.WriteLine("Choose row (0,1,2): ");
+                    row = int.Parse(Console.ReadLine());
+                    Console.WriteLine("Choose column (0,1,2): ");
+                    column = int.Parse(Console.ReadLine());
+                    if (row > 2 || row < 0 || column > 2 || column < 0) messageToUser = "Please enter valid numbers 0,1,2";
+                    else if (board[row, column] != ' ') messageToUser = "This cell is already ocuupied. Please enter another cell!";
+                    Console.WriteLine(messageToUser);
+                } while (messageToUser != "");
+                if (turnOfX) board[row, column] = 'X';
+                else board[row, column] = 'O';
+                DisplayBoard();
+            } while (!SomebodyWon()&&!IsFull());
+            if (SomebodyWon()) {
```

Comment:
if (SomebodyWon() && turnOfX) 
{
      messageToUser = "Wow! X has won!";
}
else 
{
      messageToUser = "Foooooo! O has managed to do smth!";
}

Reply:
pie messages vajag vienu tab